src/main/resources/jseont_sitemap/xml/sitemap.index.jsp

```diff
@@ -18,10 +18,11 @@
 <%--@elvariable id="workspace" type="java.lang.String"--%>
 
 <c:set target="${renderContext}" property="contentType" value="text/xml;charset=UTF-8"/>
+
 <?xml version="1.0" encoding="UTF-8"?>
 <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-        <c:if test="${renderContext.liveMode}">
-                <c:set var="siteMapPath" value="${fn:replace(currentNode.url, '.html', '')}" />
+        <c:if test="${renderContext.liveMode and renderContext.site.defaultLanguage eq renderContext.mainResourceLocale.language}">
```

Comment:
does jsp's support `and` or does that have to be `&&`?

Reply:
it supports both

Reply:
learned something new today :)